pkg/kinflate/configmapandsecret/configmap_secret.go

```diff
@@ -172,3 +174,68 @@ func validateTLS(cert, key string) error {
 	}
 	return nil
 }
+
```

Comment:
not in the PR, but i'd like to see us introduce

```
type GVKNObject map[gvkn.GroupVersionKindName]*unstructured.Unstructured
```
and start using it.  Lots of methods we could hang there.